data/io.elementary.files.appdata.xml.in.in

```diff
@@ -58,6 +58,29 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="Next" date="Future" urgency="medium">
+      <description>
+        <p>New features:</p>
+        <ul>
+          <li>Initial git plugin support</li>
+          <li>Check hicolor theme if no elementary icon before falling back to default</li>
+          <li>Follow global history setting when restoring and saving tabs</li>
+        </ul>
+        <p>Other updates:</p>
+        <ul>
+          <li>Asynchronous undo/redo</li>
```

Comment:
What does this mean for users? More reliable? Performance improvements?

Reply:
I guess this can be absorbed into the general "code improvements" entry. It is part of work Corentin is doing to move file operations to Vala.  In the long run it should make the code faster and more reliable but it is not linked to any particular issue atm.

Reply:
It can probably be absorbed into the general "code improvement" entry - if this kind of entry is needed at all